plugins/continuous-toolbox/src/ContinuousFlyout.js

```diff
@@ -166,18 +170,20 @@ export class ContinuousFlyout extends Blockly.VerticalFlyout {
   /**
    * Add additional padding to the bottom of the flyout if needed,
    * in order to make it possible to scroll to the top of the last category.
-   * @param {!Blockly.utils.Metrics} metrics Default metrics for the flyout.
+   * @param {!Blockly.MetricsManager.ContainerRegion} contentMetrics Content
+   *    metrics for the flyout.
+   * @param {!Blockly.MetricsManager.ContainerRegion} viewMetrics View metrics
+   *    for the flyout.
    * @return {number} Additional bottom padding.
-   * @private
    */
-  calculateBottomPadding_(metrics) {
+  calculateBottomPadding(contentMetrics, viewMetrics) {
     if (this.scrollPositions.length > 0) {
       const lastCategory =
           this.scrollPositions[this.scrollPositions.length - 1];
       const lastPosition = lastCategory.position.y * this.workspace_.scale;
-      const lastCategoryHeight = metrics.contentHeight - lastPosition;
-      if (lastCategoryHeight < metrics.viewHeight) {
-        return metrics.viewHeight - lastCategoryHeight;
+      const lastCategoryHeight = contentMetrics.height - lastPosition;
+      if (lastCategoryHeight < viewMetrics.height) {
+        return viewMetrics.height - lastCategoryHeight;
       }
     }
     return 0;
```

Comment:
I think getMetrics_ (below this) is obsolete now?

Reply:
Yup! Forgot to remove, thanks!